irohad/ordering/impl/on_demand_connection_manager.hpp

```diff
@@ -20,24 +20,39 @@ namespace iroha {
      */
     class OnDemandConnectionManager : public transport::OdOsNotification {
      public:
+      /**
+       * Responsibilities of individual peers from the peers array
+       * Transactions are sent to three ordering services:
+       * current consumers, first and second, and previous consumer
+       * Proposal is requested from the current ordering service: issuer
+       */
+      enum PeerType {
+        kIssuer = 0,
+        kPreviousConsumer,
+        kCurrentFirstConsumer,
+        kCurrentSecondConsumer,
+        kCount
+      };
+
+      /// Collection with value types which represent peers
+      template <typename T>
+      using PeerCollectionType = std::array<T, kCount>;
+
       /**
        * Current peers to send transactions and request proposals
-       * Transactions are sent to two ordering services:
-       * current and previous consumers
-       * Proposal is requested from current ordering service: issuer
+       * @see PeerType for individual descriptions
```

Comment:
`@see`, is it https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdsee ?

Reply:
Yes.